maxim/models/dataset.py

```
@@ -1,4 +1,4 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)